ledger/accountdb_test.go

```diff
@@ -954,6 +954,15 @@ func TestCompactAccountDeltas(t *testing.T) {
 	a.Equal(sample2, data)
 }
 
+// upsertOld updates existing or inserts a new partial entry with only old field filled
```

Comment:
you can do the same with accounts' `upsertOld`

Reply:
sure.. I'll do it on a separate PR ( it goes beyond the scope of this intended change.. )